spec/unit/libraries/security_cmd_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+include MacOS
+
+describe MacOS::SecurityCommand do
+  context 'when SecurityCommand object is instantiated' do
+    it 'security should use /usr/bin/security' do
+      test = MacOS::SecurityCommand.new('/Users/vagrant/Test.p12', '')
```

Comment:
We don't need to instantiate this object for every test. You should be able to put it right inside the `describe` block and then it should be available to all the tests.

Reply:
done